examples/window/low_power.rs

```diff
@@ -61,13 +61,13 @@ fn update_winit(
     use ExampleMode::*;
     *winit_config = match *mode {
         Game => {
-            // In the default `WinitConfig::game()` mode:
+            // In the default `WinitSettings::game()` mode:
             //   * When focused: the event loop runs as fast as possible
             //   * When not focused: the event loop runs as fast as possible
```

Comment:
I think this comment is no longer accurate. Could you fix it up while you're in there?

Reply:
```suggestion
            //   * When not focused: the app will update when the window is directly interacted with
            //     (e.g. the mouse hovers over a visible part of the out of focus window), a
            //     [`RequestRedraw`] event is received, or one sixtieth of a second has passed
            //     without the app updating (60 Hz refresh rate max).
```

Reply:
How does that look? `WinitSettings::game()` and `WinitSettings::desktop_app()` both use `UpdateMode::ReactiveLowPower` in `unfocused_mode`, so I copied the comments from `Application` and just updated the refresh rate (from once per minute to 60 times per second)

Reply:
Makes sense to me.